app/views/organizations/show.html.erb

```diff
@@ -33,7 +33,7 @@
                         <div class="col-7"><% unless shift.shift_open %>Worker Name<% end %></div>
                     </div>
                     <div class="text-right">
-                        <%= link_to "Edit", edit_shift_path(shift.id), class: "btn" %>
+                        <% if shift.shift_open %><%= link_to "Edit", edit_shift_path(shift.id), class: "btn" %> <% end %>
                         <%= link_to "Delete", shift_path(shift.id), method: :delete, data: { confirm: "Are you sure?" }, class: "btn" %>
```

Comment:
I think you should wrap the `<% if %>` block around the `Delete` button as well.

Reply:
Thanks @msarit , will update